### PR TITLE
Add tools for developer configuration

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add tools for developer user whitelisting used by blockchain owners
- Add tools for developer url whitelisting used by developers

cc: @jakepyo

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1161

### CODE REVIEW

**Review by ehgmsdk20 — APPROVED** [review]
LGTM with minor comments, thanks!

File: tools/chatbot/sendCreateAppAndConfigTx.js [hunk 1]
```diff
@@ -107,7 +107,8 @@ async function processArguments() {
 }
 
 function usage() {
-  console.log('\nExample commandlines:\n  node sendCreateAppAndConfigTx.js config_local.js\n')
+  console.log("\nUsage: node sendCreateAppAndConfigTx.js <Config File>\n")
+  console.log("Example: node sendCreateAppAndConfigTx.js config_local.js 'Hello'\n")
```
[the remark this answers is not in the document]
Comment (platfowner) [reply tools/chatbot/sendCreateAppAndConfigTx.js:111]: Just updated the usage (actually it supports custom message when argv.length === 4). PTAL

File: tools/chatbot/sendMessageTx.js [hunk 2]
```diff
@@ -46,7 +46,8 @@ async function processArguments() {
 }
 
 function usage() {
-  console.log("\nExample commandlines:\n  node sendMessageTx.js config_local.js 'Hello'")
+  console.log("\nUsage: node sendMessageTx.js <Config File>\n")
```
[the remark this answers is not in the document]
Comment (platfowner) [reply tools/chatbot/sendMessageTx.js]: Updated the usage, thanks!

**Comment by platfowner** [comment]
Thanks for the review! Now merging..

File: tools/chatbot/sendCreateAppAndConfigTx.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jiyoung-an) [reply tools/chatbot/sendCreateAppAndConfigTx.js:111]: @platfowner Where can I find out 'it supports custom message when argv.length === 4'?

File: tools/developers/sendAddToDevelopersUrlWhitelistTx.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  }
+  await confirmTransaction(config.endpointUrl, timestamp, txInfo.txHash);
+}
+
+async function processArguments() {
+  if (process.argv.length !== 6) {
+    usage();
+  }
+  config = require(path.resolve(__dirname, process.argv[2]));
+  const urlKey = process.argv[3];
```
Comment (jiyoung-an) [inline tools/developers/sendAddToDevelopersUrlWhitelistTx.js:50]: question) What is URL Key?

**Review by jiyoung-an — APPROVED** [review]
LGTM with some questions!

File: tools/chatbot/sendCreateAppAndConfigTx.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (platfowner) [reply tools/chatbot/sendCreateAppAndConfigTx.js:111]: Ah, right. I'll delete it in the next PR.

File: tools/developers/sendAddToDevelopersUrlWhitelistTx.js (hunk 3 above)
Comment (platfowner) [reply tools/developers/sendAddToDevelopersUrlWhitelistTx.js:50]: it corresponds to the key in the path to the url value.
( see: http://35.194.139.219:8080/get_value?ref=/developers )

**Pull request merged**